### PR TITLE
ratatool 0.1.4 (new formula)

### PR DESCRIPTION
**Pull request by nevillelyh**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/.github/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew tests` with your changes locally?

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
> No description provided.

Please fill out the PR template, it's there for a reason, thanks.

File: Formula/ratatool.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  url "https://github.com/spotify/ratatool/releases/download/v0.1.4/ratatool-0.1.4.jar"
+  sha256 "90e9da5ac9f2d7ff96141fc5ca295b5882896501ac6d656d9211992d5541314c"
+
+  bottle :unneeded
+
+  def install
+    libexec.install "ratatool-#{version}.jar"
+    bin.write_jar_script libexec/"ratatool-#{version}.jar", "ratatool"
+  end
+end
```
Comment (MikeMcQuaid) [inline Formula/ratatool.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

**Comment by MikeMcQuaid** [comment]
This tool is not currently notable enough for inclusion: https://bot.brew.sh/job/Homebrew%20Core%20Pull%20Requests/6820/version=el_capitan/testReport/junit/brew-test-bot/el_capitan/audit_ratatool___new_formula/

I'm happy to leave this open until after it's 30 days old and see if it's looking more popular then.

**Comment by tdsmith** [comment]
In the meantime: One way you could use Homebrew to install this software is to [maintain your own tap](https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/How-to-Create-and-Maintain-a-Tap.md). The linked document has some advice; please let us know if you have any questions and we'll be happy to help you set things up.

**Comment by nevillelyh** [comment]
Thanks. We decided to keep it in our own tap.

**Pull request closed without merging**